FeedStoreChallenge/CoreDataModel/ManagedFeedImage+CoreDataClass.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  ManagedFeedImage+CoreDataClass.swift
+//  FeedStoreChallenge
+//
+//  Created by Fei Wen on 6/5/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+//
+
+import Foundation
+import CoreData
+
+@objc(ManagedFeedImage)
+public class ManagedFeedImage: NSManagedObject {
```

Comment:
`ManagedFeedImage` is an implementation detail of `CoreDataFeedStore` thus it should not be public.
Also it is not intended to be inherited from, thus it should be final.

Please remove `public` modifier and make the class `final`.
```suggestion
final class ManagedFeedImage: NSManagedObject {
```